### PR TITLE
Eslint

### PR DESCRIPTION
**Pull request by hoshin**
Applied linting rules + use of eslint instead of jshint

### CODE REVIEW

File: controllers/suggestionsController.js [hunk 1]
```diff
@@ -1,68 +1,69 @@
...
+		const newBranchName = BRANCH_PREFIX + now
+		const commitReferenceToBranchFrom = config.commitReferenceToBranchFrom
 
-    renderErrorPage(res) {
-        res.render('ko');
-    }
+		if (necessaryPullRequestDataIsDefinedAndNotEmpty(accessToken, title, description, state)) {
+			self.githubHelper().createReference(accessToken, newBranchName, commitReferenceToBranchFrom, (err) => {
+				if (!err) {
+					const testFileName = FILE_NAME_PREFIX + now + '.txt'
```
Comment (MattiSG) [inline controllers/suggestionsController.js]: > Coude be a template string  :wink:

**Comment by MattiSG** [comment]
Ok so you decided to go the no-semi way?

File: package.json [hunk 2]
```diff
@@ -38,6 +38,6 @@
 	  	"start": "./node_modules/babel-cli/bin/babel-node.js server.js",
     	"test": "mocha --compilers js:babel-core/register --recursive ./test/**/*Spec.js",
     	"qualimetry": "./node_modules/babel-cli/bin/babel-node.js node_modules/.bin/isparta cover _mocha --dir qualimetry/cover -- --compilers js:babel-core/register -R spec test/*     #&&plato -l ./.jshintrc -r -d qualimetry/platoReport -x test/ #plato broken for es6 right now",
```
Comment (MattiSG) [inline package.json]: Remove the commented-out part.

**Comment by MattiSG** [comment]
Missing a dependencies update in the `package.json`.

**Comment by hoshin** [comment]
applied linting + created .eslintrc file directly on current codebase to avoid a probable useless complicated merge

**Pull request closed without merging**